opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "1.2"
 name: "ppx_deriving_cmdliner"
-version: "0.4.0"
+version: "0.4.1"
```

Comment:
ha - didn't know about that convention. Thanks, Seb! OCaml Packaging practices are really intimidating. Glad you all write great `Makefile`s and it just works. Though, I think if I ever get to that chapter, I would like solvuu's solution. That `ml` file is much friendlier than the rest of the systems I saw.